Buildings/Fluid/Movers/Data/Fans/EnglanderNorford1992/package.mo

```diff
@@ -0,0 +1,21 @@
+within Buildings.Fluid.Movers.Data.Fans;
```

Comment:
I suggest to move this to `Buildings.Fluid.Movers.Examples.Data` as it is only used there and I don't think users will see it useful to use this particular fan. If you think the data are generally useful, then let's leave them here.

Reply:
Moved as suggested in ab1946af1893e81835011269038350cd54bead06.